lab_support/get_a_book/BookSource.md

```diff
@@ -0,0 +1,5 @@
+# Don Quixote by Miguel de Cervantes Saavedra
+
+The provided text at [./book.txt](./book.txt) was downloaded from the [Gutenberg Project](http://www.gutenberg.org/wiki/Main_Page).
```

Comment:
The text used in the lab can be found in [./book.txt](./book.txt), also [available [here](http://www.gutenberg.org/ebooks/996) on the [Gutenberg Project](http://www.gutenberg.org/wiki/Main_Page) web-site

Reply:
Done